sbtStudentCommands/IC.scala.template

```diff
@@ -0,0 +1,29 @@
+/**
+  * Copyright © 2014, 2015 Typesafe, Inc. All rights reserved. [http://www.typesafe.com]
+  */
+package sbtstudent
+
+import sbt._
+import Keys._
+import sbt.complete.DefaultParsers._
+
+import scala.Console
+import StudentKeys._
+import Zip.withZipFile
+import sbtstudent.Pssr.{getCueFolder, getRootFolder}
+
+object IC {
+  def ICselectFirstExercise: Command = Command.command("ICselectFirstExercise") { state =>
+
+    val firstExercise = state.get(mapNext).get.keys.toVector.sorted.head
+    withZipFile(state, firstExercise) { () =>
+      val currentExerciseMain = new sbt.File(getRootFolder(state), "exercises")
+      val cueFolder = getCueFolder(state)
+      val solFolder = new sbt.File(cueFolder, firstExercise)
+      IO.delete(currentExerciseMain)
+      IO.copyDirectory(solFolder, currentExerciseMain)
```

Comment:
Why do we pull the solution? Shouldn't the command simply take us to the first exercise, then give us the option to pull the solution?

The validate script, where this is used, is going to pull the solution anyway.